src/app/components/NavBar/NavBar.tsx

```diff
@@ -1,10 +1,34 @@
 import * as React from 'react';
+import { Theme, ThemeName} from "../../types";
 import './NavBar.css';
 
-export const NavBar: React.SFC = () => {
-    return(
-        <nav>
-            <a href="#" className={'nav-logo'}>Foodly</a>
-        </nav>
-    );
-};
+export type NavBarProps = {
+    themeName: ThemeName;
```

Comment:
these theme-related props don't need to be here. u should render the theme consumer in here instead